__tests__/app.test.js

```diff
@@ -163,8 +163,30 @@ expect(body).toHaveProperty("article_id", expect.any(Number))
 
 
 
-
-
+            describe('gets an array of comments back for the specified id', ()=> {
+                test('check if the array has been returned with comments', ()=> {
+   
+                 return  request(app).get('/api/articles/3/comments').expect(200).then(({body})=>{
+   
+                       expect(body.comments.length).toBe(2)
+   
+   
+                       body.comments.forEach((eachComment) => {
+   
+               expect(eachComment).toHaveProperty("comment_id", expect.any(Number))
+               expect(eachComment).toHaveProperty("votes", expect.any(Number)) 
+               expect(eachComment).toHaveProperty("created_at", expect.any(String))
+               expect(eachComment).toHaveProperty("author", expect.any(String))
+                 
+               expect(eachComment).toHaveProperty("body", expect.any(String))
+               expect(eachComment).toHaveProperty("article_id", expect.any(Number))
+                       })
+
+                       expect(body.comments).toBeSorted();
+                   })
+                })
+                     })
+   
```

Comment:
A solid happy path test ✅ Missing a bit of error testing here though 📝, we need to check:
- we get 400 when an invalid datatype is provided as the`article_id`, (`/api/articles/katherinerules/comments`)
- we get 404 when a valid but non-existant `article_id` is provided (`/api/articles/999999/comments`)
- we get 200 and an empty array when we ask for the comments from a valid existing article that has no comments (i.e. this scenario shouldn't trigger an error)